src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private int top;
+    private int bottom;
+    private int side;
```

Comment:
The variable names `top`, `bottom`, and `side` could be more descriptive to reflect that they represent lengths of the trapezoid sides. Consider renaming them to `topLength`, `bottomLength`, and `sideLength`, respectively.